### PR TITLE
Update the installation instructions for components

### PR DESCRIPTION
**Pull request by javiereguiluz**
This is based on #10971. My questions:

* Is it OK to apply this to all components?
* Is it OK to leave the Polyfills out of this change?
* If this is accepted, when upmerging to other branches, what should we do? Replace `^3.4` by each branch? In all of them except "current"? Should we use "dev-master" for master?

### CODE REVIEW

File: components/phpunit_bridge.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ Installation
 
 .. code-block:: terminal
 
-    $ composer require --dev "symfony/phpunit-bridge:*"
+    $ composer require --dev "symfony/phpunit-bridge::^3.4"
```
Comment (OskarStark) [inline components/phpunit_bridge.rst]: No double colon here and the quotes are not needed, right?

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: components/phpunit_bridge.rst (hunk 1 above)
Comment (fabpot) [reply components/phpunit_bridge.rst]: Quotes are needed by some shells like zsh IIRC.

Comment (OskarStark) [reply components/phpunit_bridge.rst]: Ok, but for now we are not using quotes across the docs, maybe we should add them everywhere then

Comment (fabpot) [reply components/phpunit_bridge.rst]: I think so. I confirm the behavior. Quoting is what I'm doing in Twig's docs: https://twig.symfony.com/doc/2.x/installation.html

**Comment by javiereguiluz** [comment]
Closing as fixed by #12347, which was a duplicate of this pull request. 🤦‍♂

**Pull request closed without merging**